modules/ROOT/pages/navigation.adoc

```diff
@@ -1,19 +1,16 @@
 = Navigation between User Interfaces
-:description: == In Bonita Portal
-
-== In Bonita Portal
-
-The default Portal profile is User. From there, the user can navigate to their other profiles. +
-image:images/UI2021.1/navigation-portal.png[Navigation Portal]
-// {.img-responsive}
+:description: == In Bonita Applications
 
 == In Bonita Applications
 
 Users may be mapped with several profiles, opening permissions to access several applications.
 Or maybe one given profile is mapped with several applications.
```

Comment:
Or maybe one given profile is mapped with several applications. => Or only one profile can be mapped with several applications.

Reply:
Done